main.go

```diff
@@ -432,3 +431,15 @@ func setStatus(s *discordgo.Session) {
 	update := discordgo.UpdateStatusData{Game: &game}
 	s.UpdateStatusComplex(update)
 }
+
+func rolesStuff(s *discordgo.Session, m *discordgo.MessageCreate) {
```

Comment:
Is MessageCreate really the best thing to use here?

Reply:
no